test/zipkin-simple.test.js

```diff
@@ -234,26 +230,26 @@ describe("Zipkin client", function () {
 					done()
 				})
 
-				Client.clientRecv(traceData, {
+				Client.sendClientRecv(traceData, {
```

Comment:
If I don't send the name on non cs, will the zipkin api accept the requests and correctly parse them?